critiquebrainz/db/revision.py

```diff
@@ -101,3 +101,61 @@ def get_votes(review_id):
             elif row.vote == True:
                 votes[revision]['positive'] += 1
     return votes
+
+
+def get_review(revision_id):
+    """Get review assosiated with a revision.
+
+    Args:
+        revision_id(uuid): ID of a revision.
+    Returns:
+        Dictionary of review related to the specified revision
+        {
+             "id": (uuid),
+             "entity_id": (uuid),
+             "entity_type": event or place or release_group,
+             "user_id": (uuid),
+             "edits": (int),
+             "is_draft": (bool),
+             "license_id": (str),
+             "language": (str),
+             "source": (str),
+             "source_url" (str)
+         }
+    """
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT review_id
+              FROM revision
+             WHERE id = :revision_id
+        """), {
+            "revision_id": revision_id
+        })
+
+        review_id = result.fetchone()[0]
+        if not review_id:
+            raise db_exceptions.NoDataFoundException("Cannot find the review for the revision id {}".format(revision_id))
+
+        # TODO: Move this to reviews when reviews are migrated.
```

Comment:
You can mark `TODO` notes with your name so that it's easier to find a person who added them if something needs to be clarified in the future.
```python
# TODO(ferbncode): ...
```